Samples/WinMLSamplesGallery/WinMLSamplesGallery/MainWindow.xaml.cs

```diff
@@ -0,0 +1,142 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml.Data;
+using Microsoft.UI.Xaml.Input;
+using Microsoft.UI.Xaml.Media;
+using Microsoft.UI.Xaml.Navigation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+// To learn more about WinUI, the WinUI project structure,
+// and more about our project templates, see: http://aka.ms/winui-project-info.
+
+namespace WinMLSamplesGallery
+{
+    /// <summary>
+    /// An empty window that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainWindow : Window
+    {
+        private readonly List<(string Tag, Type Page)> _pages = new List<(string Tag, Type Page)>
+        {
+            ("all_samples", typeof(AllSamplesPage)),
+            ("audio", typeof(Audio)),
+            ("benchmark", typeof(Benchmark)),
+            ("home", typeof(HomePage)),
+            ("image", typeof(Image)),
+            ("video", typeof(Video)),
+        };
+
+        public static Frame mainFrame;
+
+        public MainWindow()
+        {
+            this.InitializeComponent();
+            this.Title = "Windows AI Samples Gallery";
+            mainFrame = contentFrame;
+        }
+
+        private void nvSample_ItemInvoked(NavigationView sender, NavigationViewItemInvokedEventArgs args)

```

Comment:
what is the "nv" prefix of "nvSample_ItemInvoked"? can you just spell out the whole word?

Reply:
I think 'nv' is short for navigation. I think this was just the standard followed in the navigation demo for WinUI